spec/motion-spec/matchers/satisfy_spec.rb

```diff
@@ -0,0 +1,18 @@
+# -*- encoding : utf-8 -*-
+describe "Matcher::Satisfy" do
+  it 'satisfy passes when the block returns true' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.